src/styles/common.css

```diff
@@ -89,3 +93,33 @@ body {
 .features-main-row .features-right{
     padding: 0;
 }
+.navbar .login-button {
```

Comment:
Nitpick: I don't think we should be merging these newlines on the Navbar styles.